content/index.es.md

```diff
@@ -62,67 +66,70 @@ Enlaces rápidos a recursos:
+Los estándares de las Pautas de Accesibilidad para el Contenido Web (WCAG) son estables y sirven de referencia cuando son publicados como una "Recomendación del W3C", que es un estándar web.
 
--   [**Cómo cumplir con las WCAG 2**: Una guía de referencia rápida y personalizable que incluye los requisitos (criterios de conformidad) y técnicas de las Pautas de Accesibilidad para el Contenido Web](http://www.w3.org/WAI/WCAG21/quickref/) que es, en esencia, [la lista de control de WCAG 2](http://www.w3.org/WAI/WCAG21/quickref/). La mayoría de la gente utiliza esta guía de referencia rápida como herramienta principal para trabajar con las WCAG.
--   **Técnicas para las WCAG 2** ([Técnicas 2.1](https://www.w3.org/WAI/WCAG21/Techniques/), [Técnicas 2.0](https://www.w3.org/TR/WCAG20-TECHS/)) proporcionan información específica para desarrollar contenido accesible, como por ejemplo fragmentos de código HTML. Las técnicas son “informativas”, lo cual quiere decir que no tiene por qué usarlas. La base para determinar la conformidad con WCAG 2 reside en los *criterios de conformidad* del texto WCAG 2 estándar y no en las técnicas. Lea más sobre las [Técnicas en las FAQ](/standards-guidelines/wcag/faq/#techs).
--  **Comprender las WCAG 2** ([Comprender 2.1](https://www.w3.org/WAI/WCAG21/Understanding/), [Comprender 2.0](https://www.w3.org/TR/UNDERSTANDING-WCAG20/)) contiene asistencia adicional para aprender a implementar las WCAG 2 dirigida a aquellas personas que quieran comprender las pautas y criterios de conformidad en profundidad.
+* [WCAG 2.0](https://www.w3.org/TR/WCAG20/) se publicó el 11 de diciembre del 2008.
+* [WCAG 2.1](https://www.w3.org/TR/WCAG21/) se publicó el 5 de junio del 2018 y se actualizó el 21 de septiembre de 2023. Las [preguntas frecuentes sobre WCAG 2](/standards-guidelines/wcag/faq/) ofrecen más información sobre esta actualización.
+* [WCAG 2.2](https://www.w3.org/TR/WCAG22/) se publicó el 5 de octubre del 2023.
 
-Puede obtener más información sobre cómo estos documentos se relacionan entre sí consultando los **[["Documentos WCAG 2"]](/standards-guidelines/wcag/docs/)**.
+Las WCAG 2.0, 2.1 y 2.2 están diseñadas para ser compatibles con versiones anteriores, lo cual quiere decir que el contenido que cumple con WCAG 2.2 también cumple con WCAG 2.1 y WCAG 2.0. Si quiere cumplir con todas las versiones, use los recursos de WCAG 2.2 y de esta forma no tendrá que preocuparse de consultar versiones anteriores.
 
-### Traducciones
+Todos los criterios de conformidad de la versión 2.0 están incluidos en 2.1, y todos los de la versión 2.1 están en 2.2 (con la excepción de 4.1.1 que se explica en el siguiente párrafo).
 
-Las traducciones de WCAG 2 autorizadas y las no oficiales se enumeran en [["Traducciones de WCAG 2"]](/standards-guidelines/wcag/translations/).
+* WCAG 2.0 tiene 12 pautas.
+* WCAG 2.1 añade una pauta y 17 criterios de conformidad. Vea una introducción en [Novedades en WCAG 2.1](/standards-guidelines/wcag/new-in-21/es).
```

Comment:
```suggestion
* WCAG 2.1 añade una pauta y 17 criterios de conformidad. Vea una introducción en [Novedades en WCAG 2.1](/standards-guidelines/wcag/new-in-21/).
```

See https://github.com/w3c/wai-intro-wcag/pull/288/files#r1591044973 for context